relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -142,6 +168,14 @@ impl NormalizeVisitor {
         }
     }
 
+    /// Subtracts from depth and records maximum found depth.
+    ///
+    /// Use when leaving an expression.
+    fn record_depth(&mut self) {
+        self.max_depth = self.max_depth.max(self.current_depth);
+        self.current_depth = self.current_depth.saturating_sub(1);
```

Comment:
TIL `saturating_sub`. Should we `saturate_add` on line 196?

Reply:
The chance that we exhaust `u64` is very low, so I would argue that `saturating_add` is unnecessary in practice.